deploy/basic/terraform/schema.yaml

```diff
@@ -30,15 +29,6 @@ variables:
     title: "Compartment"
     description: "The compartment in which to create compute instance(s)"
 
-  availability_domain:
```

Comment:
Removed? The current command automatically get the Always Free resources AD?

Reply:
yes